pcdsdevices/lxe.py

```diff
@@ -286,18 +298,40 @@ def __init__(self, prefix='', *, egu=None, **kwargs):
             )
         super().__init__(prefix, egu='s', **kwargs)
 
-    @property
-    def _user_offset(self):
+    @user_offset.sub_value
+    def _offset_changed(self, value, **kwargs):
         """
-        Used as part of the AttributeSignal above, mirror the user offset from
-        the setpoint.
+        The user offset was changed.  Update the setpoint attribute.
         """
-        return self.setpoint.user_offset
-
-    @_user_offset.setter
-    def _user_offset(self, value):
         self.setpoint.user_offset = value
 
+    @done.sub_value
+    def	_update_position(self, value=None, **kwargs):
+        """The move was completed. Update the notepad readback."""
+        if value == self.done_value:
+            try:
```

Comment:
Now that we're repeating this block in more than a couple places, it may be worth adding as a method on the `NotepadLinkedSignal` itself - what do you think? `put_safely(value)`/`put_nofail` or something?

Reply:
Ahh, yes, that's a good idea.

Reply:
Issue created for this; will resolve it now.